ui-frontend/packages/catalog-ui-search/src/main/webapp/component/table-export/table-export.tsx

```diff
@@ -53,15 +54,15 @@ function getSrcs(selectionInterface: any) {
   return selectionInterface.getCurrentQuery().getSelectedSources()
 }
 
-function getColumnOrder(filteredAttributes: any[]): string[] {
-  return user
+function getColumnOrder(selectionInterface: any): string[] {
+  const columnOrder = user
     .get('user')
     .get('preferences')
-    .get('columnOrder')
-    .filter(
-      (property: string) =>
-        filteredAttributes.includes(property) && !properties.isHidden(property)
-    )
+    .get('inspector-summaryShown')
+    .filter((property: string) => !properties.isHidden(property))
+    const result = selectionInterface.getCurrentQuery().get('result')
+    const lazyResults = Object.values(result.get('lazyResults').results) as LazyQueryResult[]
+    return getExportAttributes(lazyResults[0], columnOrder)
```

Comment:
I think using attributes from `lazyResults[0]` will be problematic if the user exports results from multiple different products from different sources that have different attributes